build.sbt

```diff
@@ -252,7 +252,7 @@ lazy val node = (project in file("node"))
   .settings(commonSettings: _*)
   .enablePlugins(RpmPlugin, DebianPlugin, JavaAppPackaging, BuildInfoPlugin)
   .settings(
-    version := "0.9.2" + git.gitHeadCommit.value.map(".git" + _.take(8)).getOrElse(""),
```

Comment:
Maybe it would make sense to store the "0.9.3" version in a constant so that it's less probable that some unwanted change in the `git.gitHeadCommit...` gets in unnoticed?

Reply:
Sure, though I don't know what would be the good place to put it.